delivery_purchase_multi_destination/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,8 @@
+To configure delivery methods with multiple origins:
```

Comment:
This are the fragments from `delivery_multi_destination`. Here you have to put that "refer to the module ... for the configuration steps".